src/XMakeTasks/Copy.cs

```diff
@@ -298,45 +307,17 @@ private void LogDiagnostic(string message, params object[] messageArgs)
                 destinationFileExists = destinationFileState.FileExists;
             }
 
-            bool hardLinkCreated = false;
+            bool linkCreated = false;
 
-            // If we want to create hard links, then try that first
+            // If we want to create hard or symbolic links, then try that first
             if (UseHardlinksIfPossible)
-            {
-                // Do not log a fake command line as well, as it's superfluous, and also potentially expensive
-                Log.LogMessageFromResources(MessageImportance.Normal, "Copy.HardLinkComment", sourceFileState.Name, destinationFileState.Name);
+                TryCopyViaLink("Copy.HardLinkComment", MessageImportance.Low, sourceFileState, destinationFileState, ref destinationFileExists, ref linkCreated, () => NativeMethods.CreateHardLink(destinationFileState.Name, sourceFileState.Name, IntPtr.Zero /* reserved, must be NULL */));
+            else if (UseSymboliclinksIfPossible)
+                TryCopyViaLink("Copy.SymLinkComment", MessageImportance.High, sourceFileState, destinationFileState, ref destinationFileExists, ref linkCreated, () => NativeMethods.CreateSymbolicLink(destinationFileState.Name, sourceFileState.Name, SymbolicLink.File));
```

Comment:
I have set the priority to high that you see the error code if you have not the privileges to create symbolic links. I have wonder the first time why it have not create symbolic links because i have read on the error message. here is what i mean
`Low`
![image](https://cloud.githubusercontent.com/assets/19551763/17581410/eccec03c-5fa5-11e6-9be9-0677ddd55792.png)

`Normal`
![image](https://cloud.githubusercontent.com/assets/19551763/17581570/dabdf894-5fa6-11e6-915e-8d2fbe3340d7.png)

`High`
![image](https://cloud.githubusercontent.com/assets/19551763/17581606/151364f2-5fa7-11e6-886d-6ee34bba4c41.png)